custom/icds/messaging/indicators.py

```diff
@@ -115,8 +140,6 @@ class LSSubmissionPerformanceIndicator(LSIndicator):
     def __init__(self, domain, user):
         super(LSSubmissionPerformanceIndicator, self).__init__(domain, user)
 
-        child_locations = [l.location_id for l in self.user.sql_location.child_locations()]
-        self.aww_user_ids = user_ids_at_locations(child_locations)
```

Comment:
just seemed like the kind of thing that we should get from the primary data source rather than a secondary data source. we can probably get it down to one query if we remove user id from the list of emitted values [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/couchapps/users_extra/views/users_by_location_id/map.js). we could probably remove user id there since it's not giving us any extra information, and then we could query that couch view with a keys parameter